src/code/z_camera.c

```diff
@@ -526,282 +869,7416 @@ s32 Camera_GetWaterBoxBgCamSetting(Camera* camera, f32* waterY) {
     return (camSetting == CAM_SET_NONE) ? -2 : camSetting;
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_camera/func_800CC938.s")
+void func_800CC938(Camera* camera) {
+    func_800DDFE0(camera);
+}
+
+/**
+ * Calculates the angle between points `from` and `to`
+ */
+s16 Camera_CalcXZAngle(Vec3f* to, Vec3f* from) {
```

Comment:
Aren't xz planes normally walls in this coordinate system?

Reply:
They can be yes, but it's not limited to. This function is only used once in camera, and it's 2 actor positions that are fed into it, not walls